crux-bench/src/crux/bench/main.clj

```diff
@@ -11,15 +11,21 @@
   (bench/post-to-slack (format "*Starting Benchmark*, Crux Version: %s, Commit Hash: %s\n"
                                bench/crux-version bench/commit-hash))
 
-  (bench/with-nodes [node]
-    (let [devices-results (devices/run-devices-bench node)
-          weather-results (weather/run-weather-bench node)
-          watdiv-results (watdiv-crux/run-watdiv-bench node {:test-count 100})
-          result-messages [(bench/results->slack-message devices-results :ts-devices)
-                           (bench/results->slack-message weather-results :ts-weather)
-                           (bench/results->slack-message
-                            [(first watdiv-results) (watdiv-crux/->query-result (rest watdiv-results))]
-                            :watdiv-crux)]]
-      (bench/send-email-via-ses (string/join "\n" result-messages))))
+  (->> (bench/with-nodes [node]
+         (let [devices-results (devices/run-devices-bench node)
+               weather-results (weather/run-weather-bench node)
+               watdiv-results (watdiv-crux/run-watdiv-bench node {:test-count 100})
+               result-messages [(bench/results->slack-message devices-results :ts-devices)
+                                (bench/results->slack-message weather-results :ts-weather)
+                                (bench/results->slack-message
+                                  [(first watdiv-results) (watdiv-crux/->query-result (rest watdiv-results))]
+                                  :watdiv-crux)]]))
+
+       (reduce (fn [acc [node-type result]]
```

Comment:
you're not using `acc` at all here, so I'd be surprised if this included all the results?

tbh, I suspect you don't need a `reduce` anyway - `reduce` is usually used when later elements need to access the results of iterating through earlier elements, whereas in this case you can probably combine them at the end?